### ISSUE TITLE
The what & when of requests. 

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#220]
1) One thing that COS flagged immediately upon seeing our implementation of the Open Data Button is that we're not clear enough for authors on WHAT is actually required. Often you don't need the data for a whole paper, but more a certain figure, experiment or other sub-component. I think for things like the website, this level of specificity isn't needed, however, for authors it would be invaluable. It's worth thinking about how we facilitate this.

2) For both Data and Article requests, we have no concept of urgency. Is it worth considering a "this is urgent" tag we could add? 

This is related to #217 which is the why, but it's a different issue as this is the "what".

**Comment by markmacgillivray** [comment]
Having multiple data requests for one paper is a very big change to the current process - it was known when we did the open data button that this would be an issue, and that potentially we have no way to ever "close" a data request on a particular paper - it is always possible that more or different data may be wanted by a different user, and we do not know when all data for a given paper is exhausted. It is not a simple definition like "I got this paper / did not get this paper" and even if it was "I got data X" and there is no more data, that would not stop someone believing there is or should be a data Y and thinking they have the right to ask for it. Also there is very limited capability to actually find data related to a paper yet. So this one needs strategic decision beyond technical decision. Unassigning for now, as could not be progressed without further discussions.
